crates/sovereign-sdk/module-system/sov-modules-stf-blueprint/src/lib.rs

```diff
@@ -9,7 +9,9 @@ mod tx_verifier;
 use std::marker::PhantomData;
 
 pub use batch::Batch;
-use borsh::BorshSerialize;
+use borsh::{BorshDeserialize, BorshSerialize};
+use rs_merkle::algorithms::Sha256;
+use rs_merkle::MerkleTree;
```

Comment:
Why not jmt?

Reply:
because our sequencer commitments use a regular merkle tree